src/db/config/config.js

```diff
@@ -24,5 +26,6 @@ module.exports = {
     host: process.env.HOST,
     url: process.env.DATABASE_URL,
     dialect: 'postgres',
+    logging: false
```

Comment:
Logging should be allowed for `production`

Reply:
`logging`option that disables Sequelize logging has been taken out in development mode